charts/cloudzero-agent/README.md

```diff
@@ -79,124 +130,51 @@ prometheus-node-exporter:
   enabled: true
 ```
 
-This will deploy the necessary resources for metric scraping.
-
-#### Passing values to subcharts
-
-Values can be be passed to the metrics exporter subcharts by adding values according to the values specification for [kube-state-metrics](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml) and [prometheus-node-exporter](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/values.yaml) in the `kube-state-metrics` and `prometheus-node-exporter` sections respectively.
-
-A common addition may be to pull the container images from custom image registries/repositories:
```

Comment:
I think it's fine to remove this. I would hope that users would know to do this anyway. but, just noting that it was put in here because some users were tripped up because they needed to set these values, but didn't know to do so.

having said that, I feel it's safe to assume some level of familiarity with helm subcharts and their own environment, so I'm okay with removing it